dev/NavigationView/NavigationView.cpp

```diff
@@ -688,14 +700,46 @@ void NavigationView::UpdateTopNavRepeatersItemSource(const winrt::IInspectable&
     m_topDataProvider.SetDataSource(items);
 
     // rebinding
+    UpdateTopNavRepeaterItemsSource(items);
```

Comment:
>UpdateTopNavRepeaterItemsSource [](start = 4, length = 31)

This name is so close to the calling funtions name. Would it be possible to differentiate them a bit more?  RebindTopNavRepeaterItemsSource

Reply:
To keep it using the same naming scheme as UpdateTopNavOverflowRepeaterItemsSource(), an alternative might be: UpdateTopNavPrimaryRepeaterItemsSource

(The item source we pass to the repeater is obtained by calling `m_topDataProvider.GetPrimaryItems()` (hence the Primary term suggestion).

Reply:
That suggestion seems good

Reply:
Done.